### ISSUE TITLE
ListBlobs operation does not set next marker correctly

### ISSUE THREAD

**Issue by AleksandrIakhnev** [arafato/azurite#142]
## Issue
The returned `marker` in `ListBlobs` operation response  is always `0` 

## Stopper
It is not possible to paginate across results with more than 5000 items

##Example
(container has 1000 blobs):
`curl http://localhost:10000/devstoreaccount1/test_container?restype=container&comp=list&maxresults=2`

Response:

```xml
<EnumerationResults ServiceEndpoint="http://localhost:10000/devstoreaccount1" ContainerName="test_container">
<Prefix/>
<Marker>0</Marker>
<MaxResults>2</MaxResults>
<Blobs>
....  first and two blobs 
</Blobs>
</EnumerationResults>
```

**Comment by arafato** [comment]
@AleksandrIakhnev confirmed as bug. Markers haven't really be implemented yet. Fix is coming very soon in v1.8.2

**Comment by arafato** [comment]
@AleksandrIakhnev just released v1.8.2 which bring support for markers in ListBlobs

**Comment by AleksandrIakhnev** [comment]
@arafato That was really quick fix! Thank you! I really appreciate your good work.

**Comment by arafato** [comment]
@AleksandrIakhnev there was an error in the implementation (see #144). Just released v1.8.3 which should fix it now.

**Comment by AleksandrIakhnev** [comment]
@arafato Thank you for letting me know
